src/otx/core/model/entity/classification.py

```diff
@@ -80,6 +80,33 @@ def reset_explain_hook(self) -> None:
         """Clear all history of explain records."""
         self.explain_hook.reset()
 
+    def get_custom_forward(self):
+        from otx.algo.hooks.recording_forward_hook import ReciproCAMHook
```

Comment:
get_forward_with_explain()? Or we can add forward_with_explain() as an abstract method and suppose we always have it. If so, we don't need to pass it to the exporter as a parameter.
Does that impl work for all cls models?